<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Extraction Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="ExtractURI">
        <property name="toolTip">
         <string>Extract URI as tokens representing Links?</string>
        </property>
        <property name="text">
         <string>Extract URIs as link tokens?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ExtractNonLetter">
        <property name="toolTip">
         <string>Should non letter words be used.</string>
        </property>
        <property name="whatsThis">
         <string>When analysing a document, it is possible to use for the description only words containing letters.</string>
        </property>
        <property name="text">
         <string>Extract Non-letter tokens?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Token Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="Filtering">
        <property name="toolTip">
         <string>Specify if the word filtering is enabled.</string>
        </property>
        <property name="whatsThis">
         <string>The plug-in can filter the words that are used to index the documents based on several criteria.</string>
        </property>
        <property name="text">
         <string>Apply token rules?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupFiltering">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="_3">
           <item>
            <widget class="QLabel" name="textLabel1_2">
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="text">
              <string>Maximum consecutive character occurences</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="MaxConsecutiveOccurs"/>
           </item>
           <item>
            <spacer name="spacer6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_4">
           <item>
            <widget class="QLabel" name="textLabel2_2">
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="text">
              <string>Minimum letters/characters ratio</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="NormalRatio"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_5">
           <item>
            <widget class="QLabel" name="textLabel1_3">
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="text">
              <string>Maximum number of non-letter characters</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer6_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="MaxNonLetter"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Filtering</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupFiltering</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractNonLetter</sender>
   <signal>toggled(bool)</signal>
   <receiver>Filtering</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractNonLetter</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupFiltering</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
